setup.py

```diff
@@ -1,11 +1,12 @@
+from __future__ import absolute_import
 try:
     from setuptools import setup
 except ImportError:
     from distutils.core import setup
 
 setup(
     name='fakecouch',
-    version='0.0.10',
+    version='0.1.0',
     description='Fake implementation of CouchDBKit api for testing purposes',
```

Comment:
I can't find a definitive answer, but I think the way to indicate that this package supports both Python 2 and 3 is to add classifiers:
```py
    classifiers=[
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 3",
    ],
```
If some point releases are supported but not others that can be indicated too:
```py
         "Programming Language :: Python :: 2.7",
```